questions.md

```diff
@@ -0,0 +1,10 @@
+# Module 3 - Questions
+
+## Pull Requests
+1. Is it possible to limit permissions on a per branch basis?
+
+2. Who can perform a merge from a pull request?
```

Comment:
Different for [individual user accounts](https://help.github.com/articles/permission-levels-for-a-user-account-repository/) and [organization accounts](https://help.github.com/articles/repository-permission-levels-for-an-organization/).